core/storage/base_model/gae_models.py

```diff
@@ -419,6 +447,13 @@ class BaseCommitLogEntryModel(BaseModel):
     # The version number of the model after this commit.
     version = datastore_services.IntegerProperty()
 
+    @staticmethod
+    def get_export_method():
+        """The history of commits is not relevant for the purposes of
```

Comment:
Is that the reason? I thought that the reason why we are excluding it was because the intention of Takeout was to allow migration to other platforms, and this does not inhibit that?

Reply:
Regardless, once we come up with a conclusion on why we don't include it, I will update the docstring

Reply:
Has this been addressed?

Reply:
This has not been addressed.

Reply:
Let's leave it as-is for now. I've tracked it in the spreadsheet and we can ask @BenHenning about it when he's back.